framework/framework.md

```diff
@@ -24,6 +24,39 @@ The following terms are used in this document with a specific meaning. Definitio
 * **target equipment**: [[IEC]](https://std.iec.ch/terms/terms.nsf/0/B9C576C31B4E488DC125771F002F58E9?OpenDocument) the equipment, machinery, apparatus or plant for which *safety* is to be achieved
 * **target system**: The *safety-related system* (specific or assumed) under consideration. For the purposes of this document, this is assumed to include a *component* incorporating software based on *Linux*
 
+### ISO 26262 Definitions
+
+* **architecture**: The representation of the structure of the *system* that allows identification of building blocks, their boundaries and interfaces, and includes the allocation of requirements to those building blocks.
+* **assessment**: The examination of whether or not a *system* achieves the safety objectives.
+* **audit**: The examination of an implemented process with regard to the process objectives.
+* **baseline**: The version of an approved set of one or more work products that serves as a basis for change.
+* **component**: A non-system level *element* that is logically or technically separable and is comprised of more than one hardware part or one or more software units. (TODO - Normalize this definition with the component above)
+* **configuration data**: The data that is assigned during element build and that controls the element build process. (TODO - Reword to make more comprehensible)
+* **element**: A system, hardware or software component, hardware part, or software unit.
+* **error**: The discrepancy between a computed, observed or measured value or condition, and the true, specified or theoretically correct value or condition.
+* **failure**: The termination of an intended behaviour of an element due to the manifestation of a fault.
+* **fault**: An abnormal termination that can cause an *element* to fail.
+* **fault tolerance**: The ability to deliver a specified functionality in the presence of one or more specified *faults*.
+* **fault tolerant time interval**: The minimum time-span from the occurrence of a *fault* to the possible occurrence of a hazardous event, if the safety mechanisms are not activated. (TODO -- Is there a better or more generic terminology for this?)
+* **inspection**: The examination of work products, following a formal procedure, to detect safety anomalies.
+* **latent fault**: A fault whose presence is not detected by a safety mechanism nor perceived by the driver within an appropriate time.
+* **malfunctioning behavior**: A *failure* or unintended behavior of an *element* with respect to its design intent.
+* **modified condition/decision coverage**: (MC/DC) A code coverage metric that indicates the percentage of single condition outcomes that independently affect a decision's outcome.  In addition, it expects that all code blocks and execution paths have been tested.
```

Comment:
I'm not convinced that it is necessary to include this, as it is a specific and unambiguous term